### DISCUSSION TITLE
Student copilot not working in vscode

### DISCUSSION THREAD

**Discussion by Sof2222** [community/community#107980] [Copilot Conversations]
### Select Topic Area

Bug

### Body

- note posted as a comment elsewhere and told to make a new discussion - 

I am a student and in github itself it says I have copilot access but when I am in vscode it says 'get started with githubcopilot start 30-day free trial'.. I am unsure why that is? (note attached the pictures)
I have tried uninstalling githubcopilot as well as toggling the 'manage trusted extentions' on it under the profile part of vs code + restarted a few times but nothing is fixing it.. Note I have been using it pretty consistently for a while now, so is not a 'new set up' issue
Any suggestions?
(Note I was having a different error this morning where I was getting this error: "copilot Cannot read properties of undefined (reading 'split')" most of the time when trying to use copilot, so updated it and lost access after I updated it).
<img width="661" alt="Screenshot 2024-02-12 at 09 44 44" src="https://github.com/community/community/assets/131628600/904b7a22-3220-46e9-9278-a8b5be1f3ad3">

![Screenshot 2024-02-12 at 09 44 49](https://github.com/community/community/assets/131628600/702ff527-d199-42d5-a903-405c0eeb70e6)

**Comment by 1tte** [comment]
maybe you can repeat installing the extension and log in again with a github account that already has co pilot, for more details, you can go here https://stackoverflow.com/questions/71806576/why-is-my-github-copilot-not- working-all-of-a-sudden

**Comment by Sof2222** [comment]
Uninstalling and reinstalling didnt do anything... I also logged in and out and restarted vs code multiple times. Not it doesn't even come up under 'trusted extentions' now after the reload too now..

**Comment by Sof2222** [comment]
Just found another thread looking with the same issue:
https://github.com/orgs/community/discussions/58082

Seems like it is not just a student issue but that I am using 2 accounts (one connected to copilot - the student account and another connected to my github account so I can push etc. - my private account which I had before I got the student account, so have everything saved there).
Seeing as it is duplicated I will close this one and post to the other discussion
